### PR TITLE
Adding

### PR DESCRIPTION
**Pull request by Suvarna-Thejas-CE**
## Overview

Updating documentation to:
1. Add help for finding ppt access token. 
2. Adding link to MyDataHelpsUI
3. Adding information about why to embed within MyDataHelps.
4. Restructuring the examples of what is in the repo. 

Wondering if we should get rid of the "if you want to edit a component, because it's not helpful, but i think it might be an actual use case. 

Adding other logistics stuff like Code of Conduct and Contributing

## Security

REMINDER: All file contents are public.

- [X ] I have ensured no secure credentials or sensitive information remain in code, metadata, comments, etc.
  - [X ] Please verify that you double checked that .storybook/preview.js does not contain your participant access key details. 
  - [X ] There are no temporary testing changes committed such as API base URLs, access tokens, print/log statements, etc.
- [X ] These changes do not introduce any security risks, or any such risks have been properly mitigated.

only adding links to other public sites. 

### Testing
Tested in markdown previewer.

### Documentation
- [ ] If this feature requires a developer doc update, tag @CareEvolution/api-docs.

Consider "Squash and merge" as needed to keep the commit history reasonable on `main`.

## Reviewers

Assign to the appropriate reviewer(s). Minimally, a second set of eyes is needed ensure no non-public information is published. Consider also including:
- Subject-matter experts
- Style/editing reviewers
- Others requested by the content owner

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[![License](https://img.shields.io/github/license/CareEvolution/MyDataHelpsStarterKit)](https://github.com/CareEvolution/MyDataHelpsStarterKit)
+
+## How to contribute to MyDataHelpsUI
```
Comment (chrisnowak) [inline CONTRIBUTING.md:3]: Looks like these should be updated

**Comment by chrisnowak** [comment]
Just one comment, otherwise 🚢

File: CONTRIBUTING.md (hunk 1 above)
Comment (Suvarna-Thejas-CE) [reply CONTRIBUTING.md:3]: what do you think they need to be updated to? They are pointing at StarterKit?

Comment (chrisnowak) [reply CONTRIBUTING.md:3]: It says "How to contribute to MyDataHelpsUI"...presumably it should be "How to contribute to MyDataHelps Starter Kit"?

Comment (chrisnowak) [reply CONTRIBUTING.md:3]: (Here and right below)

**Pull request merged**